### PR TITLE
fix: Password input box width greater than login card

### PR DESCRIPTION
**Pull request by iamdevvalecha**
Fixes #
https://github.com/fossasia/open-event-frontend/issues/7155
#### Short description of what this resolves:
resolves ui part of password box

#### Changes proposed in this pull request:
just width part fixed
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamdevvalecha** [comment]
The Changes:-
Screenshot from 2021-05-14 22-13-44

File: app/templates/components/forms/login-form.hbs [hunk 1]
```diff
@@ -34,7 +34,7 @@
           <div class="field">
             <div class="ui left icon input">
               <i class="lock icon"></i>
-              <Input class="rounded-r-none" @type="{{if this.showPass 'text' 'password'}}" @name="password" placeholder={{t "Password"}} @value={{this.password}} />
+              <Input class="rounded-r-none" style="width:30%" @type="{{if this.showPass 'text' 'password'}}" @name="password" placeholder={{t "Password"}} @value={{this.password}} />
```
Comment (ranjitkshah) [inline app/templates/components/forms/login-form.hbs:37]: use classes

Comment (iamdevvalecha) [reply app/templates/components/forms/login-form.hbs]: okay

**Review by ranjitkshah — DISMISSED** [review]

**Comment by MeghalBisht** [comment]
Now the issue is not solved. It was solved in the previous commit

**Comment by iamdevvalecha** [comment]
> Now the issue is not solved. It was solved in the previous commit

re fixing the previous commit.

**Comment by iamdevvalecha** [comment]
@MeghalBisht check please?

File: app/templates/components/forms/login-form.hbs (hunk 1 above)
Comment (MeghalBisht) [inline app/templates/components/forms/login-form.hbs:37]: class

Comment (iamdevvalecha) [reply app/templates/components/forms/login-form.hbs:37]: okay

**Comment by iamdevvalecha** [comment]
sending new pr

**Pull request closed without merging**